LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -235,9 +235,21 @@ size_t bufferCopyAndSanitize(char* dst, char* src)
 ///
 size_t formatProcessLog(char* chBuf)
 {
-    // {"Source":"Process","LogEntry":{"Logline":"<chBuf>"},"SchemaVersion":"1.0.0"}
-    const char* prefix = "{\"Source\":\"Process\",\"LogEntry\":{\"Logline\":\"";
-    const char* suffix = "\"},\"SchemaVersion\":\"1.0.0\"}\n";
+    LoggerSettings settings;
+    const char* prefix;
+    const char* suffix;
+    if (Utility::CompareWStrings(settings.LogFormat, L"JSON"))
+    {
+        // {"LogEntry":{"Source":"Process","Logline":"<chBuf>"},"SchemaVersion":"1.0.0"}
+        prefix = "{\"LogEntry\":{\"Source\":\"Process\",\"Logline\":\"";
+        suffix = "\"},\"SchemaVersion\":\"1.0.0\"}\n";
+    }
+    else {
+        // <LogEntry><Source>Process</Source><Logline><chBuf>Z</Logline></LogEntry>
+        prefix = "<LogEntry><Source>Process</Source><Logline>";

```

Comment:
nit: same here as in XML, we'll need to come back and agree on the schema unification between XML and JSON.